### ISSUE TITLE
COP Mission 5-3🐛Mobs untargetable yet attacking players

### ISSUE THREAD

**Issue by Arnd-SN** [DerpyProjectGroup/topaz#4810]
Mobs in Pso'Xja for COP Mission 5-3 sometimes appear invisible, and will attack players and remain untargetable.

It appears that the more players/pets that are in zone increase the odds that this will happen. I went in zone with 2 other players, and I was able to view all mobs, though some did appear to "disappear" at a shorter distance then they should. For example, a pot would disappear at around 28 yalms if it wasn't targeted but would disappear at 50 if it was.

From testing done on a vanilla LSB server, it appears that once there are 6+ entities in zone the zone appears to become a little unstable (mobs phasing in and out), with 8+it becomes completely unusable. 

Unsure if this has anything to do with recent LSB changes.

**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my issue will be ignored.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened, and I have checked the commit log to see if the issue has been resolved since my server was last updated.

## Branch affected by issue 

`base`

## Steps to reproduce

Go into zone with 6+ player entites/pets and mobs begin phasing in/out randomly, and can become untargetable yet will attack players.

In the pictures, wasn't able to see lizard in room until it was directly on top of us.

![fillmode no lizard](https://github.com/LandSandBoat/server/assets/66708123/52959ef7-be90-45bf-96d6-7b54804cca23)

![lizard](https://github.com/LandSandBoat/server/assets/66708123/3319ce00-345e-4f2d-876d-0aef8ee426e1)

## Expected behavior

Mobs should remain seen and not phase in and out, should also remain targetable for all players.

**Comment by UmeboshiXI** [comment]
I think I recall ASB or maybe Horizon having this issue at some point. I can't seem to be able to find the bug report at this time though. I'll link it here if I come across it.

**Comment by Arnd-SN** [comment]
So this is specifically in regards to Tenzen's path. I just looked in the LSB discord, and I found a discussion about it, but it doesn't appear that there was any resolution.

**Comment by WinterSolstice8** [comment]
I spent several days working on just this issue with the help of atom0s and there was no obvious reason that we could come up with for this to happen

tl;dr: we just don't know how to fix it right now, sorry.

**Comment by Xaver-DaRed** [comment]
> I just looked in the LSB discord

LSB doesn't have a discord.

**Comment by Arnd-SN** [comment]
Sorry, you're correct, I meant the FFXiPrivate Servers discord.

**Comment by WinterSolstice8** [comment]
Does this still happen after the merge of #5043 ? Retail is still a bit different -- but this workaround might be sufficient for now.
